IDEAS/Examples/IBPSA/TwoWayOverwriteTRV.mo

```diff
@@ -0,0 +1,231 @@
+within IDEAS.Examples.IBPSA;
```

Comment:
Can you inline this component in `IDEAS/Examples/IBPSA/SingleZoneResidentialHydronic.mo` since it should not be used elsewhere?

Reply:
Sorry I'm not sure what you mean by inline. I think the best would be to create a base classes package under Examples.IBPSA and insert it in there. That highlights the fact that shouldn't be used somewhere else.